smarts/core/bubble_manager.py

```diff
@@ -233,28 +233,21 @@ def in_airlock(pos_):
         )
 
         transition = None
+        # XXX: When a travelling bubble disappears and an agent is airlocked or
+        #      hijacked. It remains in that state.
         # TODO: Depending on step size, we could potentially skip transitions (e.g.
         #       go straight to relinquish w/o hijacking first). This may be solved by
         #       time-based airlocking. For robust code we'll want to handle these
         #       scenarios (e.g. hijacking if didn't airlock first)
-        if (
```

Comment:
Can you give an explanation of why deleting `in_*(prev_pos)` would work? Sorry, I didn't understand the problem when looking at the code

Reply:
This delta only works if the vehicle is moving. If the vehicle is stationary and the bubble is travelling the delta is 0.